client/app/views/clientView.js

```diff
@@ -53,6 +53,15 @@ define(['jquery', 'controllers/clientController', 'controllers/tableController',
     function createResizeSensors(tables) {
       tables.forEach(function (t) {
         var div = $('#' + t.rootElement.id);
+
+        var width = $(window).width();
```

Comment:
createResizeSensors is never called, so this code never gets triggered

Reply:
It's called on line 84?

Reply:
It's called on line 84?

Reply:
Are we looking at the same line?
```//createResizeSensors(tables); THIS FUNCTION BREAKS THINGS I THINK! -IRA```